src/assets/index.html

```diff
@@ -3,16 +3,22 @@
 <head>
     <meta charset="UTF-8">
     <title>yarr!</title>
-    <link rel="stylesheet" href="./static/stylesheets/bootstrap.min.css">
-    <link rel="stylesheet" href="./static/stylesheets/app.css">
-    <link rel="icon" href="./static/graphicarts/favicon.svg" type="image/svg+xml">
-    <link rel="alternate icon" href="./static/graphicarts/favicon.png" type="image/png">
-    <link rel="manifest" href="./manifest.json" />
+    <link rel="stylesheet" href="./static/stylesheets/bootstrap.min.css" crossorigin="use-credentials">
+    <link rel="stylesheet" href="./static/stylesheets/app.css" crossorigin="use-credentials">
+    <link rel="icon" href="./static/graphicarts/favicon.svg" type="image/svg+xml" crossorigin="use-credentials">
+    <link rel="alternate icon" href="./static/graphicarts/favicon.png" type="image/png" crossorigin="use-credentials">
```

Comment:
are these required? what's the purpose of the additional tags?

Reply:
`crossorigin="use-credentials"` is needed if we use Basic Authentication of HTTP. It is an edge case where I deploy yarr using the HTTP server's Basic Authentication (Caddy in this case). The browser is able to load `manifest.json` correctly, but would not be able to load other files specified by `manifest.json`.